src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -226,36 +233,37 @@ private void AnalyzeDirectory(DirectoryInfo directoryPath, FileInfo configuratio
                 Console.WriteLine(Environment.NewLine + Environment.NewLine + $"Directory: {directoryPath}");
 
                 int numOfSuccesses = 0;
-                using (IReportWriter reportWriter = this.GetReportWriter(reportFormat.ToString(), outputFilePath, directoryPath.FullName))
+               
+                var filesFailed = new List<FileInfo>();
+                foreach (FileInfo file in filesToAnalyze)
                 {
-                    var filesFailed = new List<FileInfo>();
-                    foreach (FileInfo file in filesToAnalyze)
+                    int res = AnalyzeTemplate(file, null, configurationsFilePath, reportFormat, outputFilePath, runTtk, verbose, false, reportWriter, false, logger);
+                    if (res == 0)
                     {
-                        int res = AnalyzeTemplate(file, null, configurationsFilePath, reportFormat, outputFilePath, runTtk, verbose, false, reportWriter, false, logger);
-                        if (res == 0)
-                        {
-                            numOfSuccesses++;
-                        }
-                        else if (res == 1)
-                        {
-                            filesFailed.Add(file);
-                        }
+                        numOfSuccesses++;
                     }
+                    else if (res == 1)
+                    {
+                        filesFailed.Add(file);
+                    }
+                }
 
-                    Console.WriteLine(Environment.NewLine + $"Analyzed {numOfSuccesses} file(s).");
-                    if (filesFailed.Count > 0)
+                Console.WriteLine(Environment.NewLine + $"Analyzed {numOfSuccesses} file(s).");
+                if (filesFailed.Count > 0)
+                {
+                    var errorMessage = $"Unable to analyze {filesFailed.Count} file(s):";
+
+                    foreach (FileInfo failedFile in filesFailed)
                     {
-                        logger.LogError("Unable to analyze {numFilesFailed} file(s):", filesFailed.Count);
-                        foreach (FileInfo failedFile in filesFailed)
-                        {
-                            logger.LogError("\t{failedFile}", failedFile);
-                        }
+                        errorMessage += $" {failedFile}";
                     }
+
+                    logger.LogError(errorMessage);
```

Comment:
``` suggestion
                    logger.LogError($"Unable to analyze {filesFailed.Count} file(s): {string.Join(",", filesFailed)}";
```

Reply:
This doesn't really display very well.  Some of the lines shown here as removed were already removed as part of your PR, so this isn't an accurate representation of the real proposed change.